backend/src/event/Event.ts

```diff
@@ -22,20 +22,39 @@ export class EventBasicCt extends EventBasicUserId {
   amount: decimal
 }
 
+export class EventBasicCtX extends EventBasicCt {
+  xUserId: number
+  xCommunityId: number
+  contributionId: number
+  amount: decimal
+}
+
 export class EventBasicRedeem extends EventBasicUserId {
   transactionId?: number
   contributionId?: number
 }
 
+export class EventBasicCtMsg extends EventBasicCt {
+  message: string
```

Comment:
Sorry, i overlooked this the last time.
Please rename the message attribut to messageId and change the type string to type number
We don't want to overload the Eventprotocol with business content, only the identifiers are necessary to get the content for analysis or statistics

Reply:
okay, done
thought we were handling the string itself